docs/src/_contribute/prerequisites.md

```diff
@@ -55,15 +55,13 @@ This means that you're ready to start Meltano CLI development. For API and UI de
 
 <div class="notification is-warning">
   <p><strong>Metrics (anonymous usage data) tracking</strong></p>
-  <p>As you contribute to Meltano, you may want to disable <a href="/reference/settings#send-anonymous-usage-stats">metrics tracking</a> globally rather than by project. You can do this by setting the environment variable `MELTANO_DISABLE_TRACKING` to `True`:</p>
+  <p>As you contribute to Meltano, you may want to disable <a href="/reference/settings#send-anonymous-usage-stats">metrics tracking</a> globally rather than by project. You can do this by setting the environment variable `MELTANO_SEND_ANONYMOUS_USAGE_STATS` to `False`:</p>
 <pre>
 # Add to `~/.bashrc`, `~/.zshrc`, etc, depending on the shell you use:
-export MELTANO_DISABLE_TRACKING=True
```

Comment:
Confirming this is going to be added back in https://github.com/meltano/meltano/issues/3429

Reply:
The understanding I got from the various discussions is that a `disable_tracking` setting would only be added for maintaining backwards compatibility for existing users, and so should not be a recommended way to configure the `send_anonymous_usage_stats` setting going forward. Adding it back would defeat that purpose _unless_ we intend to remove `send_anonymous_usage_stats` in favour of `disable_tracking` as the single setting in a future release. I am not sure if that has been decided, but either way I think we should only document one of the two options to make deprecation of the other as smooth as possible for new users 🙂

Reply:
@kgpayne good point. I think we should add it to #5975 and #3434 for sure though 😄 thanks for clarifying!